manifests/init.pp

```diff
@@ -55,6 +55,10 @@
     notify  => Service['dnsmasq'],
   }
 
+  file { '/etc/sysconfig':
+    ensure => 'directory',
+  }
+
```

Comment:
Is this really necessary? Dir should be in place already.

```% rpm -qf /etc/sysconfig
filesystem-3.2-25.el7.x86_64
```

Reply:
No. Dismissing...